scripts/alert_emails/utils.ts

```diff
@@ -88,6 +89,7 @@ function generateAlertEmailContent(
     img_url: `${thermometerBaseURL}/therm-${newLevel}-${oldLevel}.png`,
     last_updated: lastUpdated,
     location_url: `${locationURL}?utm_source=risk_alerts&utm_medium=email`,
+    subscribe_link: `https://covidactnow.org/alert_signup?utm_campaign=risk_alert_forward_subscribe&utm_source=risk_alerts&utm_medium=email`,
```

Comment:
Added this "risk_alert_forward_subscribe" campaign in case we want to track this in analytics later...